src/OrbitQt/orbitmainwindow.ui

```diff
@@ -270,6 +270,13 @@ QPushButton:disabled {
             </property>
            </widget>
           </item>
+          <item row="1" column="0">
+           <widget class="OrbitSamplingReport" name="inspectionReport" native="true">
+            <property name="visible">
+              <bool>false</bool>
+            </property>
+           </widget>
+          </item>
```

Comment:
What does this correspond to?

Reply:
This is meant to be in the next pr for changing the sampling tab, not this one; thanks for flagging. Removed